### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Grammar correction

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The new internal linking feature for Elasticsearch allows you to link to other d
 
 ## Transformations
 
-We have added a new **Merge on time** transform that can combine many time series or table results. Unlike the join transform this combines the result into one table even when the time values does not align / match.
+We have added a new **Merge on time** transform that can combine many time series or table results. Unlike the join transform this combines the result into one table even when the time values do not align / match.
```
Comment (achatterjee-grafana) [inline docs/sources/guides/whats-new-in-v7-1.md:55]: Add a comma - "Unline the join transform,"

Comment (NiteshKoushik) [reply docs/sources/guides/whats-new-in-v7-1.md]: Hello @achatterjee-grafana , changes made as per the suggestion

**Review by achatterjee-grafana — APPROVED** [review]
Thanks!

**Review by achatterjee-grafana — APPROVED** [review]
Looks good.

**Pull request merged**